### PR TITLE
add onSuccess and onFail callback

### PR DESCRIPTION
**Pull request by lama-pacos**
add api onSuccess and onFail

### CODE REVIEW

File: src/Component.js [hunk 1]
```diff
@@ -15,13 +17,28 @@ export const CopyToClipboard = React.createClass({
...
+      options
+    } = this.props;
+
     const elem = React.Children.only(children);
 
-    copy(text, options);
+    const result = copy(text, options);
+
     if (onCopy) {
       onCopy(text);
```
Comment (nkbt) [inline src/Component.js]: I would just do `onCopy(text, result)` instead, and keep API small

Comment (lama-pacos) [inline src/Component.js]: now?

File: README.md [hunk 2]
```diff
@@ -118,6 +118,7 @@ Text to be copied to clipboard
 Optional callback, will be called when text is copied
 
 
+
```
Comment (nkbt) [inline README.md]: Probably worth adding short note to the readme too

Comment (lama-pacos) [inline README.md]: It that ok?

**Comment by lama-pacos** [comment]
sorry, can you publish, please?

**Comment by nkbt** [comment]
```
+ react-copy-to-clipboard@4.3.0
✔
```

Cheers!

**Pull request merged**